ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala

```diff
@@ -42,6 +43,20 @@ trait RTS {
     }
   }
 
+  /**
+   * WARNING: UNSAFE - convert between IO[?, A] and Future - do not use this method.
+   * Seriously, try not to.
+   */
+  final def unsafeToFuture[E, A](io: IO[E, A]): concurrent.Future[E \/ A] = {
```

Comment:
your lack of naming conviction is unfortunate.

Reply:
I wanted to! But this makes writing scalafix rules easier...

Reply:
are you just forbidding methods named `unsafe*`?